examples/cli-example/src/main/java/io/getunleash/example/AdvancedConstraints.java

```diff
@@ -8,8 +8,8 @@
 public class AdvancedConstraints {
     public static void main(String[] args) throws InterruptedException {
         UnleashConfig config = UnleashConfig.builder().appName("client-example.advanced")
-            .customHttpHeader("Authorization", "default:development.5ad8536b3f4f527355a92a628b4519751804d499ac7155794eb039bf")
-            .unleashAPI("https://app.unleash-hosted.com/hosted/api").instanceId("example")
+            .customHttpHeader("Authorization", "*:development.5f4b073484dd6de99fc2c8a9667b457e6cdb850224ffd71c32996aea")
```

Comment:
Is this an intentional change? I ask because every time I work on an SDK I change this equivalent code for local dev purposes and often forget to roll it back.